lib/spack/spack/build_systems/autotools.py

```diff
@@ -86,7 +106,7 @@ def do_patch_config_guess(self):
                 check_call([my_config_guess], stdout=PIPE, stderr=PIPE)
                 # The package's config.guess already runs OK, so just use it
                 return True
-            except:
+            except Exception:
```

Comment:
Why did this change to "except Exception", isn't that what plain "except" means?